.github/dependabot.yml

```diff
@@ -56,6 +56,12 @@ updates:
             - "🤖 Dependencies"
         schedule:
             interval: "weekly"
+    -   package-ecosystem: "gomod"
+        directory: "/kv/" # Location of package manifests
```

Comment:
The `kv` is really generic name and it should be specifized. Maybe you can change it to `cloudflarekv`

Reply:
Agree it. I'll change.